src/evergreen/api.py

```diff
@@ -79,6 +78,7 @@ def __init__(
         timeout: Optional[int] = None,
         session: Optional[requests.Session] = None,
         log_on_error: bool = False,
+        configure_logging: bool = True,
```

Comment:
Should we rename this parameter to `use_default_logger_factory` or something like that, since we are only setting the `logger_factory` parameter in `structlog.configure()` if it's `True`? It doesn't stop users from setting other parameters, right?